src/components/music/RollChannel.tsx

```diff
@@ -46,7 +46,8 @@ interface NoteRenderCoordinates {
 const Wrapper = styled.div<WrapperProps>`
   position: absolute;
   top: 0;
-  margin: 0 10px;
```

Comment:
Not sure if this is the best solution. But the scrollbars were jumping when the mouse was moving over the roll grid under certain conditions depending on the window size (reported https://discord.com/channels/554713715442712616/570925559346102273/945034223675408494). 

With vertical margin the bottom of the roll doesn't look very nice, but it seems to fix the issue :/